pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -158,9 +177,18 @@ func (r *Reconciler) reconcile(ctx context.Context, source *v1alpha1.ApiServerSo
 		r.Logger.Error("Unable to create the receive adapter", zap.Error(err))
 		return err
 	}
-
 	// Update source status
 	source.Status.MarkDeployed()
+
+	// Only create EventType for Broker sinks.
+	if source.Spec.Sink.Kind == "Broker" {
```

Comment:
I still don't agree with this Broker test

Reply:
Will create an issue for it so that we can track it later on.
We are doing this everywhere. I think for this release we should leave it, and then probably do the change.

Reply:
https://github.com/knative/eventing/issues/1202